File: Exercice 1/main.java
// Pas réussi


public class Playground
{
	public static void main(String[ ] args) {
	    Book lempiredesanges = new Book("L'empire des anges");
	    System.out.println(Book.getName());
	}
}

class Book {
    public String title;
    public String price;

    Book(String title, String price) {
        this.title = title;
        this.price = price;
    }

    Object getName() {
        return this.title;
        return this.price;
    }
}

File: Exercice 1/main.php
<?php
class Book {
  public $title;
  public $price;

  function __construct($title, $price) {
    $this->title = $title;
    $this->price = $price;
  }
}

$jaiChoisiLaVie = new Book("J'ai choisi la vie", 6);
print_r($jaiChoisiLaVie);

?>
